cms/static/require-config.js

```diff
@@ -27,6 +27,7 @@ require.config({
         "jquery.immediateDescendents": "coffee/src/jquery.immediateDescendents",
         "datepair": "js/vendor/timepicker/datepair",
         "date": "js/vendor/date",
+        "moment": "js/vendor/moment.min",
```

Comment:
@AkA84 those are definitely good questions! I'm going to let @singingwolfboy take a crack at them; he wrote a lot of the javascript in Studio and has a better understanding of it than this Python developer. He was out all last week, so I'll try to see if he can jump into this today - but it may be a day or two as he catches up on things.